src/components/PersonLink.tsx

```diff
@@ -0,0 +1,67 @@
+import { Link } from 'react-router-dom';
+import classNames from 'classnames';
+
+import { Person } from '../types';
+
+interface Props {
+  person: Person;
+  activeSlug: string | undefined;
+  search: string;
+}
+
+export const PersonLink: React.FC<Props> = ({
+  person: {
+    slug,
+    sex,
+    name,
+    born,
+    died,
+    motherName,
+    fatherName,
+    mother,
+    father,
+  },
+  activeSlug,
+  search,
+}) => {
+  return (
+    <tr
+      data-cy="person"
+      className={classNames({ 'has-background-warning': activeSlug === slug })}
+    >
+      <td>
+        <Link
+          to={{ pathname: `/people/${slug}`, search }}
+          className={classNames({ 'has-text-danger': sex === 'f' })}
+        >
+          {name}
+        </Link>
```

Comment:
As a code style suggestion, consider moving the `Link` component properties to separate variables. This would make your JSX cleaner and easier to read.